src/main/java/frc/robot/maps/subsystems/LedMap.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.maps.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+
+public class LedMap {
+    public AddressableLEDBuffer ledBuffer;
+    public AddressableLED led;
+
+    public LedMap() {
+        this.led = new AddressableLED(0);
+        this.ledBuffer = new AddressableLEDBuffer(0);
+    }
+
+    public LedMap(AddressableLED led, AddressableLEDBuffer ledBuffer) {
```

Comment:
It would be kind of nice if we could pass in just the port number and buffer size, and have the map create it automatically?